### ISSUE TITLE
Tracking issue for Viewing Mapping priority fixes

### ISSUE THREAD

**Issue by philbarker** [t3-innovation-network/desm#478]
This links to individual issues that should be solved around viewing mapping, summarized in this [screen cast](https://www.pjjk.net/temp/DESMVids/simplescreenrecorder-2024-03-11_14.15.37.mkv) which looks at the Demo & Testing profile on https://tool.desmsolutions.org/mappings-list

The priority issues are 
- [ ] Who has access #473 
- [ ] Too many specs are listed #454 (and should list specs not orgs!)
- [ ] Provide a direct link to mappings #419 
- [x] Bigger buttons #455 
- [ ] Schema name and version should be shown #470 
- [x] Better display when multiple properties are mapped #426  
- [x] #418 

We should also discuss the following which have no specific issue:
- Moving the mapping predicate indicator so that it is between spine and spec properties
- Tightening display / removing white space
- Decide on specification Vs schema, and be consistent

These issues seem either not to be so important, or need to wait for other things to be fixed
- #405 (fix elsewhere when doing transforms, and it'll not display here)
- #404 (needs considering alongside transforms
- #417 (need to figure out what it means)
- #447 (not actually part of the public view; would need new icons)

**Comment by jeannekitchens** [comment]
@edgarf @philbarker the Microcredential Mapping is a real example of a mapping where the public page, is difficult to use https://desm.credentialengine.org/mappings-list.  Select the Microcredentail Mapping   First we have to start with how a person navigates to this page.  The page is not on the main menu.  Users think they should select "View Mappings" from the main menu but that is only for mappers.  Rather, user selects "View Shared Mappings" from the body of the home page.  The description that accompanies the button needs to be reworded.  It says "View Specification" and " To see crosswalks currently in process.  People are viewing mapping that have been set to a status of active.  They may be in process or completed. 
![image](https://github.com/t3-innovation-network/desm/assets/20917206/31fca117-0fbe-4c13-b64c-33368f0619a0).

Other page issues:

1. The name of the mapping provided via the configuration needs to show on the top of the page.
2. The description of the mapping via the configuration needs to show on the top of the page.
3. A mapping "updated"date needs to be visible at the top of the page. Otherwise, how would anyone know how recent or dated a mapping is? 
4. When the mapping is printed or downloaded to JSON to a CSV. The name, description, and last updated date need to be included with the data. 
5.  The abstract classes are all defined.  When a person selects an abstract class.  that context needs to be clear and the definition of the absract class selected needs to be displayed. Need to be sure the abstract class defintions are included with downloaded data. 
6. Make sure the Sort Spine By functions properly and the options are relevant and work.   Not sure what Organization or Spine Class/Type mean.  When the abstract class is selected, isn't the mapping that is viewed all for that class?  Is Organization supposed to sort by the name of the organization? 
![image](https://github.com/t3-innovation-network/desm/assets/20917206/d476e79b-d88f-4d8b-b069-69ec0afe1e43)
7. Item 3 above applies to the Sorty Aligned Items By.
8. Are export options a separate issue?  The public page needs to include the JSON and CSV export options.  People are going to expect to be able to download a CSV with all of the mappings not just one at a time.  E.g., first column is the spine, second column is predicates, third column is the properties mapped to the 1st colum, 4th column is predicates, 5th column is the next schemas properties mapped to the spined and so forth.  It would be good to get the UI updates done and follow with the export options.

**Comment by philbarker** [comment]
Thank you @jeannekitchens ! I think we might need a mock-up of what the page should look like, we can start that as a new issue. I wouldn't want it to delay this work, but if we can get one together in time it should be done with this work for the sake of efficiency.

We do have an separate tag for [Data download](https://github.com/t3-innovation-network/desm/issues?q=is%3Aissue+is%3Aopen+label%3A%22Data+Output%22) which has an issue #424 for public facing download. I think there is more to be done before we can add a button to this page.
